wiki/en/Running-a-Server.md

```diff
@@ -18,50 +18,37 @@ permalink: "/wiki/Running-a-Server"
+**No**. You can use the Servers listed by the built in Directories and use Jamulus without running a Server or choose a third party hosting service. If you just want an undisturbed session, use the [soloing technique described on the Tips and Tricks page](tips-tricks-more#have-a-undisturbed-session-on-any-server).
 
 While setting up a Server isn't difficult, it's a good idea to read the following backround information to avoid some problems:
 
 ### Speed and latency
 
-Many people attribute problems to the Server that are in fact problems with the _Client_. Much depends on the Clients' hard/software setup, the networks that _they_ are on, and whether they are sticking to [Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
-
 **_The capability of the Server itself (and the network it's on) is NOT the main determinant of the quality of a Jamulus session!_**
 
-If you plan to be playing regularly with the same people, **you are strongly advised** to first make sure that each member of the group is set up to use Jamulus properly. Do this by finding a Public Server with a reasonable ping time for all of you (20 ms or less perhaps), all connect to that and work to fix any individual issues (verifying that they can [follow Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) in particular). Use the solo technique above to prevent being interrupted if needed.
-
-Once any issues with musicians have been solved in this way, you can then investigate hosting your own Server either at home or on a cloud host such as Amazon, which may result in better latency than Servers run at home.
-
-### Bandwidth – do you have enough?
-
-Unless you plan on hosting more than about 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up), then you are unlikely to run out of bandwidth. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
-
-### In general
+Usually, problems are on the _Client_ side and should be fixed there. Have a look at the [Troubleshooting page](/wiki/Client-Troubleshooting) if needed.
+Various problems can arise when setting up Servers - especially when run on a low-bandwidth home connection.
+Therefore:
+* Use other peoples' (public) Servers at first to isolate any issues that may in fact be due to Client configuration.
+* Ensure you have enough bandwidth: Less than 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up) are fine. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
 
-- Consider using a cloud host to get better ping times if you're having problems
+### General notes
 
-- Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
+* Consider using a cloud host, not your home internet connection, to get better ping times if you're having problems
```

Comment:
This _is_ answering "Do I need to run a Server?", so it belongs before the "Speed and Latency" section.